TedTalks/TedTalks/ViewController.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ViewController.swift
+//  TedTalks
+//
+//  Created by Miguel Angel Askar Rodriguez on 3/08/21.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
```

Comment:
You can remove this given it is not being used

Reply:
Ok, removed